lib/fake_artist/table.ex

```diff
@@ -182,10 +189,17 @@ defmodule FakeArtist.Table do
     # we'll batch some of the "near-real-time" updates, like collaborative drawing,
     # for performance reasons
     :timer.send_interval(@tick_timer_in_ms, :update_tick)
-    FakeArtistWeb.Endpoint.broadcast("table:#{table_name}", "update", state)
+
+    FakeArtistWeb.Endpoint.broadcast(
+      "table:#{table_topic}",
+      "update",
+      Map.delete(state, :pids_to_ids)
+    )
 
     IO.inspect(state)
 
+    :ok = FakeArtist.Hostess.forget_table_pid(self())
```

Comment:
this prevents more people from joining AND allows the hostess to re-use the name